config/dev.exs

```diff
@@ -21,6 +21,8 @@ config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
     ]
   ]
 
+config :elixir_boilerplate, ElixirBoilerplateWeb, data_identity_provider: ElixirBoilerplateWeb.DataIdentifier.TestID
```

Comment:
Could we move this to `config/config.exs` and only overwrite it in `config/prod.exs`?